mlir/docs/MIOpen/tensor_layout.md

```diff
@@ -0,0 +1,81 @@
+#Layout with group
+
+##tensorflow
+###NHWC
+<br/>
+<br/>
+####When you want to add group in tensorflow,
+####you split on dim of c or k , and tf.concat outputs 
+<br/>
+#####axis=3 split on c dim ,input layout become [N H W G C]
+input_groups = tf.split(axis=3, num_or_size_splits=groups, value=x)
+<br/>
+Example of split:
+if nhwk is 
+2,2,2,2
+[[[[1],[2]],[[3],[4]]],[[[5],[6]],[[7],[8]]]],[[[[9],[10]],[[11],[12]]],[[[13],[14]],[[15],[16]]]]
+=>
+<br/>
+after split to 2 groups :
+2,2,2,1
+2,2,2,1
+[[[[1]],[[3]]],[[[5]],[[7]]]],[[[[9]],[[11]]],[[[13]],[[15]]]]
+[[[[2]],[[4]]],[[[6]],[[8]]]],[[[[10]],[[12]]],[[[14]],[[16]]]]
+<br/>
+#####user format is [YXGCK], not final format
+weights = tf.get_variable('weights', shape=[filter_height,     
+                                                    filter_width,
+                                                    input_channels/groups,
+                                                    num_filters])
```

Comment:
`[YXGCK]` and here `tf.get_variable()` create a 4d tensor? where to parse in the `groups` information to the `weights` variable?

Reply:
groups is not real dim ,this sentence just showed original dim before split